<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="apples-MacBook-Pro.local" name="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY" tests="43" failures="6" timestamp="18 Jul 2019 10:59:36 GMT" time="415.138" errors="0">
  <testcase name="gettingDashboard" time="2.117" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="40.516" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.citi.mobileAutomation.assertEngine.Assert_Engine.assertMapping(Assert_Engine.java:13)
	at com.citi.mobileAutomation.dataMapping.Action_Mapping.actionMapping_Mobile_Application_Native_Android(Action_Mapping.java:103)
	at com.citi.mobileAutomation.dataMapping.ObjectGroups.actionOnElements(ObjectGroups.java:46)
	at com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY.gettingDashboard(Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY.java:25)
	at sun.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </failure>
  </testcase> <!-- gettingDashboard -->
  <testcase name="gettingDashboard" time="10.279" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="10.947" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="41.056" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.citi.mobileAutomation.assertEngine.Assert_Engine.assertMapping(Assert_Engine.java:13)
	at com.citi.mobileAutomation.dataMapping.Action_Mapping.actionMapping_Mobile_Application_Native_Android(Action_Mapping.java:103)
	at com.citi.mobileAutomation.dataMapping.ObjectGroups.actionOnElements(ObjectGroups.java:46)
	at com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY.gettingDashboard(Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY.java:25)
	at sun.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </failure>
  </testcase> <!-- gettingDashboard -->
  <testcase name="gettingDashboard" time="1.374" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="1.600" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="2.307" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="6.797" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="1.979" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="2.116" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="1.840" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="1.677" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="0.255" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="1.835" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="40.721" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.citi.mobileAutomation.assertEngine.Assert_Engine.assertMapping(Assert_Engine.java:13)
	at com.citi.mobileAutomation.dataMapping.Action_Mapping.actionMapping_Mobile_Application_Native_Android(Action_Mapping.java:103)
	at com.citi.mobileAutomation.dataMapping.ObjectGroups.actionOnElements(ObjectGroups.java:46)
	at com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY.gettingDashboard(Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY.java:25)
	at sun.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </failure>
  </testcase> <!-- gettingDashboard -->
  <testcase name="gettingDashboard" time="1.691" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="1.979" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="40.300" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.citi.mobileAutomation.assertEngine.Assert_Engine.assertMapping(Assert_Engine.java:13)
	at com.citi.mobileAutomation.dataMapping.Action_Mapping.actionMapping_Mobile_Application_Native_Android(Action_Mapping.java:103)
	at com.citi.mobileAutomation.dataMapping.ObjectGroups.actionOnElements(ObjectGroups.java:46)
	at com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY.gettingDashboard(Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY.java:25)
	at sun.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </failure>
  </testcase> <!-- gettingDashboard -->
  <testcase name="gettingDashboard" time="41.743" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.citi.mobileAutomation.assertEngine.Assert_Engine.assertMapping(Assert_Engine.java:13)
	at com.citi.mobileAutomation.dataMapping.Action_Mapping.actionMapping_Mobile_Application_Native_Android(Action_Mapping.java:103)
	at com.citi.mobileAutomation.dataMapping.ObjectGroups.actionOnElements(ObjectGroups.java:46)
	at com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY.gettingDashboard(Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY.java:25)
	at sun.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </failure>
  </testcase> <!-- gettingDashboard -->
  <testcase name="gettingDashboard" time="6.536" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="1.599" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="11.125" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="1.823" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="1.622" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="51.794" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.citi.mobileAutomation.assertEngine.Assert_Engine.assertMapping(Assert_Engine.java:13)
	at com.citi.mobileAutomation.dataMapping.Action_Mapping.actionMapping_Mobile_Application_Native_Android(Action_Mapping.java:103)
	at com.citi.mobileAutomation.dataMapping.ObjectGroups.actionOnElements(ObjectGroups.java:46)
	at com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY.gettingDashboard(Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY.java:25)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </failure>
  </testcase> <!-- gettingDashboard -->
  <testcase name="gettingDashboard" time="4.493" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="1.683" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="4.799" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="4.247" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="4.603" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="1.968" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="1.408" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="2.342" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="7.412" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="1.683" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="1.058" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="8.258" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="2.091" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="1.792" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="26.614" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="3.016" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
  <testcase name="gettingDashboard" time="10.043" classname="com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY"/>
</testsuite> <!-- com.citi.mobileAutomation.test_SG.Test_Apac_iOS_Externalcc_Adhoc_FCY_LCY -->
